modules/post-installation-adding-nutanix-failure-domains-compute-machines-replace.adoc

```diff
@@ -0,0 +1,223 @@
+
+. List the compute machine sets in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
++
+.Example output
+[source,text]
+----
+NAME                            DESIRED   CURRENT   READY   AVAILABLE   AGE
+<original_machine_set_name_1>   1         1         1       1           55m
+<original_machine_set_name_2>   1         1         1       1           55m
+----
+
+. Note the names of the existing compute machine sets.
+
+. Create a YAML file that contains the values for your new compute machine set custom resource (CR) by using one of the following methods:
+
+** Copy an existing compute machine set configuration into a new file by running the following command:
++
+[source,terminal]
+----
+$ oc get machineset <original_machine_set_name_1> \
+  -n openshift-machine-api -o yaml > <new_machine_set_name_1>.yaml
+----
++
+You can edit this YAML file with your preferred text editor.
+
+** Create a blank YAML file named `<new_machine_set_name_1>.yaml` with your preferred text editor and include the required values for your new compute machine set.
++
+If you are not sure which value to set for a specific field, you can view values of an existing compute machine set CR by running the following command:
++
+[source,terminal]
+----
+$ oc get machineset <original_machine_set_name_1> \
+  -n openshift-machine-api -o yaml
+----
++
+--
+.Example output
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+  name: <infrastructure_id>-<role> <2>
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+        machine.openshift.io/cluster-api-machine-role: <role>
+        machine.openshift.io/cluster-api-machine-type: <role>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>
+    spec:
+      providerSpec: <3>
+        ...
+----
+<1> The cluster infrastructure ID.
+<2> A default node label.
++
+[NOTE]
+====
+For clusters that have user-provisioned infrastructure, a compute machine set can only create machines with a `worker` or `infra` role.
+====
+<3> The values in the `<providerSpec>` section of the compute machine set CR are platform-specific. For more information about `<providerSpec>` parameters in the CR, see the sample compute machine set CR configuration for your provider.
+--
+
+. Configure the new compute machine set to use the first failure domain by adding the following to the `spec.template.spec.providerSpec.value` stanza in the `<new_machine_set_name_1>.yaml` file.
```

Comment:
maybe change `adding` to `updating or adding` should be better, because if using the 1st method "** Copy an existing compute machine set configuration into a new file by running the following command", then these fields already exist , we just need to modify their values. 
And if using the 2nd method "** Create a blank YAML file named `<new_machine_set_name_1>.yaml` with your preferred text editor and include the required values for your new compute machine set.", these fields don't exist, then we should add.

Reply:
Thanks, I like the added accuracy :nerd_face: